CMake/switch/devkita64-libnx.cmake

```diff
@@ -40,7 +40,5 @@ set(CMAKE_LD "/${TOOLCHAIN_PREFIX}ld${TOOLCHAIN_SUFFIX}" CACHE INTERNAL "")
 set(CMAKE_OBJCOPY "${TOOLCHAIN_PREFIX}objcopy${TOOLCHAIN_SUFFIX}" CACHE INTERNAL "")
 set(CMAKE_SIZE_UTIL "${TOOLCHAIN_PREFIX}size${TOOLCHAIN_SUFFIX}" CACHE INTERNAL "")
 
-set(BUILD_SHARED_LIBS OFF CACHE INTERNAL "Shared libs not available")

```

Comment:
Is this relevant for 3DS?

Reply:
No, can't see it being related to 3ds. https://github.com/MrHuu/devilutionX-3ds/pull/2/commits/775b82461b9ae5113495855d650e8387f02f1ade#diff-cddd743214a162062dc29716f9655cd3
I should probably revert it.